tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -169,18 +172,8 @@ public Project ConfigureProject()
             //project.EnableResilientPackage();
 
             project.MajorUpgrade = MajorUpgrade.Default;
-            // When set to yes, WiX sets the msidbUpgradeAttributesVersionMaxInclusive attribute,
-            // which tells MSI to treat a product with the same version as a major upgrade.
-            // This is useful when two product versions differ only in the fourth version field.
-            // MSI specifically ignores that field when comparing product versions,
-            // so two products that differ only in the fourth version field are the same product and
-            // need this attribute set to "true" to be detected.
-            // Note that because MSI ignores the fourth product version field,
-            // setting this attribute to yes also allows downgrades when the first three product version fields are identical.
-            // For example, product version 1.0.0.1 will "upgrade" 1.0.0.2998 because they're seen as the same version (1.0.0).
-            // That could reintroduce serious bugs so the safest choice is to change the first three version fields and
-            // omit this attribute to get the default of no.
-            project.MajorUpgrade.AllowSameVersionUpgrades = false;
+            // Set to true otherwise RC versions can't upgrade each other.
+            project.MajorUpgrade.AllowSameVersionUpgrades = true;
```

Comment:
Did you test running exactly the same MSI?

Reply:
... or should we only do it when the current build is "RC"?

Reply:
> Did you test running exactly the same MSI?
Yes, you get the Change/Repair/Remove screen.